fuel/src/main/kotlin/com/github/kittinunf/fuel/core/Headers.kt

```diff
@@ -1,5 +1,8 @@
 package com.github.kittinunf.fuel.core
 
+import java.util.HashMap
```

Comment:
I wonder how we can build without this .... 🤔

Reply:
Why not https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-hash-map/index.html?

Reply:
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-mutable-map/index.html

Reply:
Why do we need this? We didn't need this before??